scripts/send_erc20_packet_back.js

```diff
@@ -0,0 +1,50 @@
+// This is for emitting SendPacket event directly from Axon to initial the IBC communication
+
+async function main() {
+  // First provider account.
+  const [receiver] = await web3.eth.getAccounts();
+  // Receiver should be CKB address, which is first 20 bytes of hash of sender's lock_script
+  const sender = process.env.SENDER;
```

Comment:
Sender and receiver seem to be backward.

Reply:
why, axon cannot be the source chain, so it always gets the receiver role, not the sender role

Reply:
In `check_balance_and_send_back`, it's:

```js
  await token.approve(transfer.address, 499, {
    from: receiver,
  });
  await transfer.sendTransfer(denom, 499, sender, port, channel, 0, {
    from: receiver,
  });
```

Sender and receiver are flipped.